README.md

```diff
@@ -135,6 +135,13 @@ You can additionally add the `--days` argument to only include shows from the N
 python ruvsarpur.py --sid 18457 --days 7  -o "c:\videos\ruv"
 ```
 
+# Portable mode
+This script stores, by default, all of its config files in the current user home directory in a folder named '.ruvsarpur'. The `--portable` command line option changes this and makes the script store all configuration files in the directory you run it from.
```

Comment:
Consider simplifying the second line of this text to be more instructional, e.g. "Use the `--portable` argument to force the script to save configuration files in the current working directory instead.